### DISCUSSION TITLE
Show user activity in GitHub profile in mobile application

### DISCUSSION THREAD

**Discussion by m3r1v3** [community/community#19280] [Mobile] [answered]
In GitHub on mobile platform still no user activity in profile. It would be useful feature

**Comment by Iron-Ham** [comment, marked as answer]
Hi @merive, thank you for the suggestion. To clarify, are you looking for personal user activity that appears on user profile screens such as this:

![Screen Shot 2022-06-23 at 3 30 26 PM](https://user-images.githubusercontent.com/3388381/175382527-85f2d849-eefe-4dc3-99ee-aa3cb6e73452.png)

I've created a ticket to track that internally! If you're looking for the type of activity you may see on the Home Screen, that is now available on the Explore tab in the mobile application.
